modules/RedisTimeSeries/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM redislabs/redistimeseries

```

Comment:
`redisedge` image cannot be configured with Unix sockets?

Reply:
When using `redisedge` image, I ran into [this issue](https://github.com/redis/redis/issues/7344)
```
Opening Unix socket: bind: Address already in use
```
On investigation it turned out that the `redisedge` image was last published [2 years ago](https://hub.docker.com/r/redislabs/redisedge) built on top of Redis 5. So I decided to change to the `redistimeseries` image which contains latest Redis (v6.x) and is anyway the only Redis module that we need. (RedisEdge image additionally contains RedisAI and RedisGears modules)